030_app_L259.js

```
@@ -0,0 +1,276 @@
+app.get("/searchinbookingpage/:searchtext?",(req,res)=>{
+// {fact rule=improper-input-validation@v1.0 defects=1}
+
+    console.log(req.params);
+
+    if(req.params.searchtext!=undefined){
+
+        var search = new RegExp(req.params.searchtext, 'i');
+        var searchcond = {uname :search};
+    }
+    else{
+        var searchcond = null;    
```

Biome (2.1.2)

[error] Shouldn't redeclare 'searchcond'. Consider to delete it or rename it.

'searchcond' is defined here:


(lint/suspicious/noRedeclare)